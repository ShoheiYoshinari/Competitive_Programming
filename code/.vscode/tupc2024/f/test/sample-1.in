3
101 1000 10
1101 10111010 0
1000110 1110011011101 100011110010
