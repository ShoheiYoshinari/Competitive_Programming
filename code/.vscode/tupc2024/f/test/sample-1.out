10
0
11100100001101111010100
